src/main/java/de/tum/cit/aet/core/domain/constants/DocumentType.java

```diff
@@ -0,0 +1,17 @@
+package de.tum.cit.aet.core.domain.constants;
+
+import lombok.Getter;
+
+@Getter
```

Comment:
```suggestion
@Getter
@AllArgsConstructor
```
I would suggest adding @AllArgsConstructor annotation to the Enum, in order to reduce the boilerplate code from the constructor.